sdk/storage/azure-storage-common/test/ut/connection_reuse_test.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "test_base.hpp"
+
+#include <azure/storage/blobs/block_blob_client.hpp>
+
+#include <unordered_map>
+
+namespace Azure { namespace Storage { namespace Test {
+
+  class ConnectionReuseTest : public StorageTest {
+  };
+
+  void updateHitCountPerServerMap(
+      const Azure::Core::CaseInsensitiveMap& headers,
+      std::unordered_map<std::string, int32_t>& hitCountPerServerMap)
+  {
+    // The third part of a storage request id means the server node id.
+    // For Example, RequestId:3bcf963b-601e-0054-1f40-910c39000000, '0054' is the server node which
+    // served this request.
+    std::string serverId = headers.find("x-ms-request-id")->second.substr(14, 4);
+    if (hitCountPerServerMap.count(serverId) == 0)
+    {
+      hitCountPerServerMap[serverId] = 1;
+    }
+    else
+    {
+      ++hitCountPerServerMap[serverId];
+    }
+  }
+
+  // If connection is reused, the requests with the same connection should hit the same sever. So
+  // this test verifies whether a series of requests hit the same server.
+  TEST_F(ConnectionReuseTest, IsConnectionReused_LIVEONLY_)
```

Comment:
On a second though, I still think you should move this case into blob test project. So that you can reuse the code to setup and cleanup containers.